### PR TITLE
Homepage for Apple Clone

### PR DESCRIPTION
**Pull request by Haroonabdulrazaq**
- It includes 3 major section

1. The Hero section
2. The Grid Section
3. The Footer section
- I have fixed all Lints error I could find
Tested on Html validator and Stickler

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
```
Comment (lucasmazo32) [inline index.html:1]: - [ ] Some of the items seem a little bit off. For instance, the navbar looks like it is thrown to the left. Maybe this has to do with the size of the screen. One piece of advice is that you use a max-width property on the body, so everything is with a specific size.

> image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              />
+            </a>
+          </p>
+        </div>
+      </section>
+      <div class="grid">
+        <div class="grid-1"></div>
+        <div class="grid-2"></div>
+        <div class="grid-3"></div>
+        <div class="grid-4"></div>
```
Comment (lucasmazo32) [inline index.html]: - [ ] Please change this div tags for anchor tags (`<a></a>`). In the original webpage, they are links to another part of the page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          <li><a href="#">Environment</a></li>
+          <li><a href="#">Job Opportunities</a></li>
+          <li><a href="#">Contact Us</a></li>
+        </ul>
+      </div>
+      <span class="contact">
+        Copyright © 2014 Apple Inc. All rights reserved.
+        <a href="#">Terms of Use </a> <a href="#">Privacy</a> Policy</span
+      >
+    </footer>
```
Comment (lucasmazo32) [inline index.html]: - [ ] If you could find a font closer to the original page that would be great! Also, please align the text in the left so they both start at the same point.

> image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved kindly implement the changes below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @Haroonabdulrazaq, great job :+1: Congratulations on fixing all the changes recommended by the previous TSE :smile: On this note, I hereby approve your project :100: 

You can go ahead and merge to master
Do not forget to submit the project completion form
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Haroonabdulrazaq** [comment]
Thank you

**Pull request merged**